hamster_gtk/preferences/preferences_dialog.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, parent, app, initial, *args, **kwargs):
                 ('tmpfile_path', (_('_Temporary file'), ComboFileChooser())),
             ]))),
             (_('Miscellaneous'), LabelledWidgetsGrid(collections.OrderedDict([
-                ('autocomplete_activities_offset', (_("Autocomplete Activities Offset"),
+                ('autocomplete_activities_range', (_("Autocomplete Activities Range"),
```

Comment:
While I agree that it may be slightly better it is still not really great. At some point I think some additional (optional/hover...) description would be useful for most settings I guess. I suggest we revisit the naming choices then instead. Also, this may have the benefit of ``hamster-gtk`` having a bigger audience that may help coming up with better descriptions.